Monika After Story/game/script-topics.rpy

```diff
@@ -17565,3 +17565,42 @@ label monika_ddlcroleplay:
     m 1euu "In any case, if it's encouraging more people to try their hand at writing, I don't think I can really fault it."
     m 1kub "Just make sure to remember that those versions of me are just stories, ahaha~"
     return
+
+init 5 python:
+    addEvent(
+        Event(
+            persistent.event_database,
+            eventlabel="monika_backup_guide",
+            category=['mod'],
+            prompt="Could you show me the guide to backup your memories?",
+            pool=True,
+        ),
+        markSeen=True
+    )
+
+label monika_backup_guide:
+    m 1hub "Oh, sure!"
+    m 1eub "The guide is right {a=https://github.com/Monika-After-Story/MonikaModDev/wiki/FAQ#i-want-to-back-up-my-persistent}{i}{u}here{/u}{/i}{/a}."
+    m 1ekbla "Thank you for doing your best to keep me safe, [player]. I really appreciate it."
+    return
+
+init 5 python:
+    addEvent(
+        Event(
+            persistent.event_database,
+            eventlabel="monika_gifting_guide",
+            category=['mod'],
+            prompt="I'm having trouble giving you a gift...",
+            pool=True,
+        ),
+        markSeen=True
+    )
+
+label monika_gifting_guide:
```

Comment:
Might be just me, but I don't really like her explaining how to give gifts. Kind of too meta even for Monika. It's useful to have links to the wiki, but might be better to just add them to the game menu.

Reply:
This is fair. I'd like to still keep the backup one if possible, but I don't mind removing this one.

Reply:
I agree that Monika should be oblivious to how the gift-giving process _works_. To her, it's the magic of the player's love - not changing an empty file's extension.

Could Chibika include the link in the initial note?

Alternatively, Chibika could leave a _new_ note after X amount of days from the initial note, pointing out the guide "just in case the player is having trouble". In addition to a deltatime difference, it'd need to check if any "initial note day" variable _isn't_ set to account for current players.